src/api.ts

```diff
@@ -42,14 +42,15 @@ class ApiClient {
       },
     });
 
-    this.socket = new Socket(`${this.host.replace("http", "ws")}/ws/v1`, {
-      // If we're in a non-browser environment, then fallback to longpolling
-      transport: typeof window === "undefined" ? LongPoll : window.WebSocket,
-      params: {
-        user_token: this.userToken,
-        api_key: this.apiKey,
-      },
-    });
+    if (typeof window !== "undefined") {
+      this.socket = new Socket(`${this.host.replace("http", "ws")}/ws/v1`, {
+        transport: window.WebSocket,
```

Comment:
This seems like a bigger change: Is long polling never the right move/possible in non-browser environments?

Reply:
Yeah great point – I just removed this line so it does the default phoenix behavior of websocket and falling back to long polling, which is what we want!